templates/vhost.j2

```diff
@@ -0,0 +1,32 @@
+server {
+    listen {{ item.listen | default('80 default_server') }};
```

Comment:
Should this really default to `80 default_server`? I never really use nginx but reading the docs it seems saner to default to `80`?